assets/images/Svg.js

```diff
@@ -24,6 +24,7 @@ const svg = {
   home: require("./home.svg"),
   javascript: require("./javascript.svg"),
   jquery: require("./jquery.svg"),
+  js: require("./javascript.svg"),
```

Comment:
We already have a `javascript` key, so you don't need to add this.

Reply:
I guess the idea here was to simplify the mapping between a sources extension .js, .css, .jsx .anything. In this case, simplifying by adding an additional key .js that maps to the correct svg. 

This might be a silly question, but what all file types can we have on the browser ? Just html, js, css? 

If I don't do it this way, this then becomes N number of if... checks where N is the number of different file types possible where I'd be checking `if (extension) svgName=extensionTypeFullName` eg `if (js) svgName = javascript` or we maintain a mapping from file extension to svgName in utils or something.

Reply:
Ya'll have an extension map I think I see what you are looking for now.